service/graph.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service // import "go.opentelemetry.io/collector/service"
+
+import (
+	"context"
+	"net/http"
+
+	"go.opentelemetry.io/collector/component"
+)
+
+var _ pipelines = (*pipelinesGraph)(nil)
+
+type pipelinesGraph struct{}
+
+func (g *pipelinesGraph) StartAll(ctx context.Context, host component.Host) error {
+	// TODO actual implemenation
+	return nil
+}
+
+func (g *pipelinesGraph) ShutdownAll(ctx context.Context) error {
+	// TODO actual implemenation
+	return nil
+}
+
+func (g *pipelinesGraph) GetExporters() map[component.DataType]map[component.ID]component.Component {
+	// TODO actual implemenation
```

Comment:
If you are changing this PR for any other reason, you could fix this typo :-)

```suggestion
	// TODO actual implementation
```